src/common.browser/WebsocketMessageAdapter.ts

```diff
@@ -4,6 +4,9 @@
 // Node.JS specific web socket / browser support.
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment
 // @ts-ignore
+/* eslint-disable @typescript-eslint/no-unsafe-call */
```

Comment:
This PR is suppose to address some teams security best practices?
So I find it ironic that my biggest red flag are these eslint disable "unsafe blah" directives.

Can you please provide some justification for why these are needed, and why it is safe to do so in this case?

Reply:
I'll look at what I can do to remove these disables, but the switch from es5 to es6 is what triggers the build errors in WebsocketMessageAdapter requiring these directives. es6 is stricter about enforcing eslint rules, apparently.